odl/solvers/nonsmooth/douglas_rachford.py

```diff
@@ -28,7 +28,7 @@
 __all__ = ('douglas_rachford_pd',)
 
 
-def douglas_rachford_pd(x, prox_f, prox_cc_g, L, tau, sigma, niter,
+def douglas_rachford_pd(x, f, g, L, tau, sigma, niter,
```

Comment:
We should definitely look over the names in this interface. It becomes even worse in the forward-backward example where an additional `h` pops up. Even `x` is up to discussion IMO. @ozanoktem, @hilding79, @aringh?